tests/Hedgehog.Benchmarks/Hedgehog.Benchmarks.fsproj

```diff
@@ -2,20 +2,23 @@
 
   <PropertyGroup>
     <OutputType>Exe</OutputType>
-    <TargetFramework>netcoreapp2.0</TargetFramework>
+    <TargetFramework>netcoreapp3.1</TargetFramework>
   </PropertyGroup>
 
   <ItemGroup>
     <Compile Include="Program.fs" />
   </ItemGroup>
 
   <ItemGroup>
-    <None Include="paket.references" />
+    <PackageReference Include="BenchmarkDotNet" Version="0.11.3" />
   </ItemGroup>
   
   <ItemGroup>
     <ProjectReference Include="..\..\src\Hedgehog\Hedgehog.fsproj" />
     <ProjectReference Include="..\Hedgehog.Tests\Hedgehog.Tests.fsproj" />
   </ItemGroup>
-  <Import Project="..\..\.paket\Paket.Restore.targets" />
+  
+  <ItemGroup>
+    <PackageReference Update="FSharp.Core" Version="4.2.3" />
```

Comment:
See comment above about `Update`. And perhaps we want `Version="4.*"`?

Reply:
It is ok to depend on a specific version `FSharp.Core` here because this project isn't a library project consumed by others but an executable project only used "internally" by the "Hedgehog developers" for testing.  

paket's lock file shows that it resolved version 4.6.1 for the `Test` group.  I picked version 4.2.3 because it matches the constraint imposed by Unqoute before I thought to look at the lock file.  I think picking any of these versions is fine.
https://github.com/hedgehogqa/fsharp-hedgehog/blob/2a68ee22e85b5d9026b3b21a6a0a04bce9cef086/paket.lock#L475